public/app/plugins/panel/geomap/globalStyles.ts

```diff
@@ -80,3 +81,112 @@ export function getGlobalStyles(theme: GrafanaTheme2) {
     }
   `;
 }
+
+// Open Layer styles
+export const style = new Style({
```

Comment:
these are all VERY specific to the measure tool right?  they should be enclosed within a measure specific envelope (FC/class/whatever) not golbal styles

the rest of this file is *trying* (but often failing) to make the `ol-xxx` css classes work with grafana themes